python/lsst/ip/isr/isrTask.py

```diff
@@ -716,10 +734,18 @@ def updateVariance(self, ampExposure, amp):
                               (amp.getName(), gain, patchedGain))
                 gain = patchedGain
 
+            if self.config.doEmpiricalReadNoise and overscanImage is not None:
+                stats = afwMath.StatisticsControl()
+                stats.setAndMask(overscanImage.mask.getPlaneBitMask(maskPlanes))
+                readNoise = afwMath.makeStatistics(overscanImage, afwMath.STDEVCLIP, stats).getValue()
```

Comment:
I'm assuming a plain old `.getValue()` is adequate here even though earlier uses of `makeStatistics` to get, e.g., the mean specifies `.getValue(afwMath.MEAN)`.